monitoring/pra/alerts.yaml

```diff
@@ -0,0 +1,79 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: config
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: config
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: config
+    value: artesca-data-dr
+  - name: RTO
```

Comment:
by convention, we have snake_case variable : so just `rto`

Reply:
went for `rto_alert_threshold`